### DISCUSSION TITLE
Malcolm Learning Tree

### DISCUSSION THREAD

**Discussion by mmguero** [cisagov/Malcolm#500] [Training]
The Malcolm team is always working to develop and improve resources for learning about how to deploy, configure, and use Malcolm. We've created a wiki page for the [Malcolm Learning Tree](https://github.com/cisagov/Malcolm/wiki/Learning), which organizes learning modules by category with links to the documentation and video resources associated with each topic. Let us know what you think!

**Comment by trwagner1** [comment]
Love this Seth.  I will share and shout it out again and again, I want to complement EVERYONE for the HUGE improvement in educational videos since last year.  Great work, keep that work up!

I am from an OT/Scada world, but, come into my role with 35+ years of IT experience.  The improvements in documentation and learning are tremendous as far as acceptance of having Malcolm in an environment.  Ease of use, easy to learn, all pluses in ANY business world.
